acceptance-tests/tests/src/test/java/org/hyperledger/besu/tests/acceptance/privacy/multitenancy/OnChainMultiTenancyAcceptanceTest.java

```diff
@@ -0,0 +1,697 @@
+  //  }
+  //
+  //  private void deletePrivacyGroupEnclaveStub() throws JsonProcessingException {
+  //    final String deleteGroupResponse = mapper.writeValueAsString(PRIVACY_GROUP_ID);
+  //    stubFor(post("/deletePrivacyGroup").willReturn(ok(deleteGroupResponse)));
+  //  }
+  //
+  //  private void retrievePrivacyGroupEnclaveStub() throws JsonProcessingException {
+  //    final String retrieveGroupResponse =
+  //        mapper.writeValueAsString(
+  //            testPrivacyGroup(List.of(ENCLAVE_KEY), PrivacyGroup.Type.PANTHEON));
+  //    stubFor(post("/retrievePrivacyGroup").willReturn(ok(retrieveGroupResponse)));
+  //  }
+  //
+  //  private void sendEnclaveStub(final String testKey) throws JsonProcessingException {
+  //    final String sendResponse = mapper.writeValueAsString(new SendResponse(testKey));
+  //    stubFor(post("/send").willReturn(ok(sendResponse)));
+  //  }
+  //
+  //  private void receiveEnclaveStub(final PrivateTransaction privTx) throws
+  // JsonProcessingException {
+  //    final BytesValueRLPOutput rlpOutput = getRLPOutputForReceiveResponse(privTx);
+  //    final String senderKey = privTx.getPrivateFrom().toBase64String();
+  //    final String receiveResponse =
+  //        mapper.writeValueAsString(
+  //            new ReceiveResponse(
+  //                rlpOutput.encoded().toBase64String().getBytes(UTF_8), PRIVACY_GROUP_ID,
+  // senderKey));
+  //    stubFor(post("/receive").willReturn(ok(receiveResponse)));
+  //  }
+  //
+  //  private BytesValueRLPOutput getRLPOutputForReceiveResponse(
+  //      final PrivateTransaction privateTransaction) {
+  //    final BytesValueRLPOutput bvrlpo = new BytesValueRLPOutput();
+  //    privateTransaction.writeTo(bvrlpo);
+  //    return bvrlpo;
+  //  }
+  //
+  //  private BytesValueRLPOutput getRLPOutput(final PrivateTransaction privateTransaction) {
+  //    final BytesValueRLPOutput bvrlpo = new BytesValueRLPOutput();
+  //    privateTransaction.writeTo(bvrlpo);
+  //    return bvrlpo;
+  //  }
+  //
+  //  private PrivacyGroup testPrivacyGroup(
+  //      final List<String> groupMembers, final PrivacyGroup.Type groupType) {
+  //    return new PrivacyGroup(PRIVACY_GROUP_ID, groupType, "test", "testGroup", groupMembers);
+  //  }
+  //
+  //  private static PrivateTransaction getValidSignedPrivateTransaction(final Address
+  // senderAddress) {
+  //    return PrivateTransaction.builder()
+  //        .nonce(0)
+  //        .gasPrice(Wei.ZERO)
+  //        .gasLimit(3000000)
+  //        .to(null)
+  //        .value(Wei.ZERO)
+  //        .payload(Bytes.wrap(new byte[] {}))
+  //        .sender(senderAddress)
+  //        .chainId(BigInteger.valueOf(2018))
+  //        .privateFrom(Bytes.fromBase64String(ENCLAVE_KEY))
+  //        .restriction(Restriction.RESTRICTED)
+  //        .privacyGroupId(Bytes.fromBase64String(PRIVACY_GROUP_ID))
+  //        .signAndBuild(TEST_KEY);
+  //  }
```

Comment:
Why do we have these tests commented out? We don't need them or did you forget to uncomment them? :)

Reply:
Done